Packages/Tracking/Core/Runtime/Scripts/LeapXRServiceProvider.cs

```diff
@@ -246,6 +246,20 @@ public int warpingAdjustment
             }
         }
 
+        [Tooltip("Should the device location be relative to the Main Camera of the scene.  \n\n")]
+        [SerializeField]
+        private bool _positionDeviceRelativeToMainCamera = true;
+        /// <summary>
+        /// Allows for the manual placement of the Tracking Hardware.
+        /// This device offset mode is incompatible with Temporal Warping.
+        /// This is only used if the deviceOffsetMode is 'Transform'.
+        /// </summary>
```

Comment:
I think it _is_ compatible with temporal warping and can be used without deviceOffsetMode being 'Transform' - but they will get additional offsets